pkg/localize/locales/en/cmd/registry_crud.en.toml

```diff
@@ -53,10 +53,10 @@ Delete a Service Registry instance along with all of its schema and API artifact
 [registry.cmd.delete.example]
 one = '''
 # Delete a Service Registry instance by name
-rhoas service-registry delete --name <name>
+rhoas service-registry delete --name name
 
 # Delete a Service Registry instance by ID
-rhoas service-registry delete --id <id>
+rhoas service-registry delete --id id
```

Comment:
Same comment as before. You could create a Kafka instance and copy the ID value for a realistic value.